src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/ActionSidebarContent.tsx

```diff
@@ -83,7 +83,7 @@ const ActionSidebarFormContent: FC<ActionSidebarFormContentProps> = ({
           `}
           message={false}
         />
-        <div className="text-gray-900 text-md flex gap-1">
+        <div className="text-gray-900 text-md flex gap-1 break-all">
```

Comment:
do we want break-all or our custom `break-word` here?

Reply:
yes, otherwise when the content of header is too long size of sidebar is broken